qml/pages/CategoriesPage.qml

```diff
@@ -0,0 +1,110 @@
+import QtQuick 2.0
+import Sailfish.Silica 1.0
+import org.chum 1.0
+import "../components"
+
+Page {
+  id: page
+  allowedOrientations: Orientation.All
+
+  SilicaListView {
+    id: view
+    anchors.fill: parent
+
+    header: Column {
+        spacing: Theme.paddingLarge
+        width: view.width
+
+        PageHeader {
+            title: "Chum"
+            //% "Categories"
+            description: qsTrId("chum-categories")
+        }
+    }
+
+    delegate: ListItem {
+      contentHeight: Theme.itemSizeSmall
+      onClicked: pageStack.push(Qt.resolvedUrl("../pages/PackagesListPage.qml"), {
+                                    //% "%1 (packages)"
+                                    subTitleAll: qsTrId("chum-available-packages-category").arg(model.category),
+                                    //% "%1 (applications)"
+                                    subTitleApp: qsTrId("chum-available-apps-category").arg(model.category),
+                                    applicationsOnly: Chum.showAppsByDefault,
+                                    category: model.categoryIds
+      })
+
+      Label {
+        anchors {
+          left: parent.left
+          leftMargin: Theme.horizontalPageMargin
+          right: parent.right
+          rightMargin: Theme.horizontalPageMargin
+          verticalCenter: parent.verticalCenter
+        }
+        text: model.category
+        wrapMode: Text.WordWrap
+        color: parent.highlighted ? Theme.highlightColor : Theme.primaryColor
+      }
+    }
+
+    model: ListModel {
```

Comment:
Im not sure we should hard-code the categories, cant we just get them dynamically from the packages?  I get that may cause issues with translations, but, for example Advanced Camera in the package page currently says the category is "Other", but thats not listed here.  If we do it from the packages, it will always be correct.

Reply:
I think we should add Other and keep the hardcoded list. 

If we build it dynamically, we will have to figure out how to translate them and how to order. By having hard-coded list (same as in official store currently, except avoiding split between Internet and Network) we would guide developers to choose the correct one. That list is not final, we can adjust it as needed, but some kind of policy would be good to enforce.

But take time and think about pros and cons :)